lib/windows.js

```diff
@@ -6,6 +6,17 @@ var request = require('request');
 var url = 'https://lic.apps.microsoft.com/licensing/certificateserver/?cid=';
 var sigXPath = '//*//*[local-name(.)=\'Signature\' and namespace-uri(.)=\'http://www.w3.org/2000/09/xmldsig#\']';
 
+module.exports.readConfig = function (configIn) {
```

Comment:
It looks like something in my IDE has played havok with the indentation. I'll get it sorted.